x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/hooks/use_styles.ts

```diff
@@ -30,6 +30,13 @@ export const useStyles = () => {
         font-weight: ${euiTheme.font.weight.bold};
       }
     }
+    & .euiDataGrid__leftControls {
+      > .euiButtonEmpty:hover:not(:disabled),
+      .euiButtonEmpty:focus {
+        text-decoration: none;
+        cursor: default;
+      }
+    }
```

Comment:
Let's add a comment regarding what exactly are we doing here and why are we doing that

Reply:
what's unclear about this code?  Class names and pseudo-classes are clear and descriptive.

Reply:
Sometimes you come back to a piece of code after a while and it can be hard to understand why it's even there, i think a single line comment can make it easier to quickly understand why someone put that there.

```
// disabling the hover and on-click effects from unclickable elements in EuiGrid
```

If you find it redundant feel free to skip